src/modules/i18n/watchClearCity.js

```diff
@@ -6,7 +6,7 @@ import AppSettings from '../settings/AppSettings'
 
 export function * clearCity (): Saga<void> {
   const appSettings = new AppSettings()
-  yield call(appSettings.clearSelectedCity)
+  // yield call(appSettings.clearSelectedCity)
```

Comment:
Instead of commenting this out we came to an other solution:

Is it possible to call the unSubscribeToPreviousCityTopic when clearCity() is called? You can use the `contentLanguage` from the appSettings.

Reply:
Unsubscribing should be removed in `loadCityContent` then.

Reply:
Good point 👍 , let me apply it and test it to make sure everything is fine.

Reply:
After doing the change I remembered that when changing the language the clearCity will not be called which means we can not reach the `unsubscribe` method. IMO, let's keep the current solution aka *commenting clearing city*. What do you think?

Reply:
Commenting it out causes other problems in the app. The only solution would be to set another field in the app settings called `previousCity`. Do you think that would work?

Reply:
Wouldn't calling unsubscribe in `clearCity` AND `changeLanguage` work?

Reply:
@klinzo  you mean, switchContentLanguage?

Reply:
@klinzo @maxammann  I handle it as Klinzo suggested on both clearCity AND switchContentLanguage 🎉